examples-generated/ec2/vpcendpointconnectionnotification.yaml

```diff
@@ -58,8 +58,8 @@ metadata:
 spec:
   forProvider:
     acceptanceRequired: false
-    networkLoadBalancerArns:
-    - ${aws_lb.test.arn}
+    networkLoadBalancerArnRefs:
+    - name: test
```

Comment:
Please keep `examples-generated` intact, make copy to `examples` and craft a working uptest-compatible example there. Thanks!